INFO     root:driver.py:41 初始化driver异常!'NoneType' object has no attribute 'to_capabilities'
INFO     root:conftest.py:22 打开APP
INFO     root:conftest.py:24 测试完成，关闭app